lib/rubocop/cop/style/hash_conversion.rb

```diff
@@ -45,6 +45,7 @@ class HashConversion < Base
 
         def on_send(node)
           return unless hash_from_array?(node)
+          return if node.arguments.any? { |argument| argument.send_type? && argument.method?(:zip) }
```

Comment:
I don't like very much hardcoding such special cases. At the very least this should be mentioned in the docs.

Reply:
Also are there other methods that might produce some output similar to that of `zip`?